requirements.txt

```
@@ -5,6 +5,8 @@ binaryornot==0.4.4
 # Code quality
 # ------------------------------------------------------------------------------
 flake8==3.7.6
+pylint-django==0.9.4
+pylint-celery==0.3
```

Ideally, we shouldn't install this when celery isn't selected, and make it conditional:

```suggestion
{% if cookiecutter.use_celery == "y" -%}
pylint-celery==0.3
{%- endif %}
```

---

Thank you for the feedback!

Also, I'm realizing that I added the requirements to the wrong file... 😞 (should've been `{{cookiecutter.project_slug}}/requirements/local.txt` instead of `requirements.txt`)

Will make the changes soon!

---

Yes, correct. I actually missed that part 👍